sdk/program/src/sysvar/mod.rs

```diff
@@ -70,6 +78,22 @@ pub trait Sysvar:
     fn to_account_info(&self, account_info: &mut AccountInfo) -> Option<()> {
         bincode::serialize_into(&mut account_info.data.borrow_mut()[..], self).ok()
     }
+    fn get() -> Result<Self, ProgramError> {
+        let mut var = Self::default();
+        let var_addr = &mut var as *mut _ as *mut u8;
+
+        #[cfg(target_arch = "bpf")]
+        let result = Self::call_syscall(var_addr);
+        #[cfg(not(target_arch = "bpf"))]
+        let result = crate::program_stubs::sol_get_sysvar(&Self::id(), var_addr);
```

Comment:
Can we make the program_stubs signature match the BPF signature?  It's unfortunate to have to muddy the code with more `target_arch` conditionals

Reply:
Stubs follow this model in general (https://github.com/solana-labs/solana/blob/1359bceb5d6c73a7b4852156d4c8676034e2e446/sdk/program/src/program.rs#L53). What did you have in mind for redirecting non-BPF program usage of the sysvar?

Reply:
ugh ok :)